src/main/resources/data/strategy_cards/cryypter.json

```diff
@@ -0,0 +1,12 @@
+[
+    {
+        "id": "cryypter_3",
+        "initiative": 3,
+        "name": "Politics",
+        "primaryTexts": [],
```

Comment:
Ideally you would add the card's texts here, like:
[
"Clause1",
"Clause2",
"Clause2"
]